taskana-adapter-camunda-listener/src/main/java/pro/taskana/adapter/camunda/exceptions/UnsupportedDatabaseException.java

```diff
@@ -0,0 +1,13 @@
+package pro.taskana.adapter.camunda.exceptions;
+
+/**
+ * This exception will be thrown if the database name doesn't match to one of the desired databases.
+ */
+public class UnsupportedDatabaseException extends RuntimeException {
+
+  private static final long serialVersionUID = 1L;
+
+  public UnsupportedDatabaseException(String name) {
+    super("Database with '" + name + "' not found");
+  }
```

Comment:
1. The message should read Database ... not supported.
2. We should stop to concatenate Strings in messages and rather use message formatter like:
super(String.format("Database with name %s not supported", name);